index.html

```diff
@@ -0,0 +1,352 @@
+            width="10">
+        </a>
+      </div>
+      <div class="header-searchbar">
+        <form class="header-form-search">
+          <input type="text" title="Search for products, brands and more"
+          placeholder="Search for products, brands and more">
+          <span class="material-symbols-outlined search-icon">search</span>
+        </form>
+      </div>
+    </div>
+    <div class="header-rightpan">
+      <div class="header-login">
+        <a class="login-button">Login</a>
+      </div>
+      <div class="header-more">
+        <div class="more">More
+          <span class="material-symbols-outlined">expand_more</span>
+        </div>
+        <div class="more-content">
+          <a href="#"><span class="material-symbols-outlined">
+            notifications
+            </span><div class="option-name">
+              Notification Preference
+            </div>
+          </a>
+          <a href="#"><span class="material-symbols-outlined">
+            live_help
+            </span> <div class="option-name">
+              24x7 Customer Care
+            </div>
+          </a>
+          <a href="#"><span class="material-symbols-outlined">
+            moving
+            </span>
+            <div class="option-name">
+              Advertise
+            </div>
+          </a>
+          <a href="#"><span class="material-symbols-outlined">
+            download
+            </span>
+            <div class="option-name">
+              Download App
+            </div>
+          </a>
+        </div>
+      </div>
+      <div class="header-cart">
+        <span class="material-symbols-outlined cart-icon">
+          shopping_cart
+          </span>
+        <span class="cart-text">Cart</span>
+      </div>
+    </div>
+  </div>
+  <div class="category-pannel">
+    <div class="category-items">
+      <div class="item">
+        <div class="item1-image">
+          <img src="https://rukminim1.flixcart.com/flap/128/128/image/f15c02bfeb02d15d.png?q=100" alt="Top Offers" />
+        </div>
+        <div class="item1-name">Top Offers
+        </div>
+      </div>
+      <div class="item">
+        <div class="item2-image">
+          <img src="https://rukminim1.flixcart.com/flap/128/128/image/29327f40e9c4d26b.png?q=100" alt="Grocery" />
+        </div>
+        <div class="item2-name">Grocery</div>
+      </div>
+      <div class="item">
+        <div class="item3-image">
+          <img src="https://rukminim1.flixcart.com/flap/128/128/image/22fddf3c7da4c4f4.png?q=100" alt="Mobiles" />
+        </div>
+        <div class="item3-name">Mobiles</div>
+      </div>
+      <div class="item">
+        <div class="item4-image">
+          <img src="https://rukminim1.flixcart.com/flap/128/128/image/c12afc017e6f24cb.png?q=100" alt="Fashion" />
+        </div>
+        <div class="item4-name">Fashion
```

Comment:
If you applying same css to multiple `divs`, always reuse them. Ex. you can just apply `item4-name` to all the next naming `divs`